osu.Game/Skinning/Components/BeatmapAttributeText.cs

```diff
@@ -81,7 +91,26 @@ protected override void LoadComplete()
             beatmap.BindValueChanged(b =>
             {
                 updateBeatmapContent(b.NewValue);
-                updateLabel();
+                getMaxPP();
+                // updateLabel() will call by getMaxPP()
+            }, true);
+
+            currentMods.BindValueChanged(s =>
```

Comment:
handling mod changes is in scope of https://github.com/ppy/osu/pull/21585 and this pull does not adhere to the direction established therein (that both pre- and post-mod values should be available for use). so either this should be reverted and not handle mods, or this PR is hard-blocked until that one is merged